### PR TITLE
Update Before you start

### PR DESCRIPTION
**Pull request by jahnvigupta**
This PR adds more details to Before you Start section of how-to and targets issue #90 .

### CODE REVIEW

File: how-to/about-how-to.md [hunk 1]
```diff
@@ -30,18 +30,21 @@ Creating a good overview comes down to thinking about who is going to be using i
 Have you ever got halfway through a how-to, only to discover you need to go and read other documentation before you can continue?
 This section is designed to prevent this from happening in the first place.
 
+Pre-reqs aren't always about going to read something else. Sometimes they are just about dependencies like "you need an internet connection".
```
Comment (jaredmorgs) [inline how-to/about-how-to.md:33]: "Pre-reqs" should be "Prerequisites". It doesn't pay to abbreviate this here.

File: how-to/about-how-to.md [hunk 2]
```diff
@@ -30,18 +30,21 @@ Creating a good overview comes down to thinking about who is going to be using i
...
 Describe what the audience needs to know, or needs to have, before attempting this how-to.
 By stating the requirements up-front, you prevent your audience from having a bad experience with your tutorial.
+And give them useful pointers about how to get whatever they need. Link to procedures or info.
```
Comment (jaredmorgs) [inline how-to/about-how-to.md:36]: The last sentence is a bit fragmented. Try linking it to the previous sentence.

"You can include links to procedures or information, or give your audience useful pointers about how to get whatever they need."

File: how-to/about-how-to.md [hunk 3]
```diff
@@ -30,18 +30,21 @@ Creating a good overview comes down to thinking about who is going to be using i
...
 Here are some example Before you Start statements:
 
 ```
 Make sure you meet the following prerequisites before following the steps:
 
-* API credentials for the {company} v3.5 API.
+* API credentials for the {company} v3.5 API. Refer to creating your API credentials.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is conceptually.Read about the RESTful API.
```
Comment (jaredmorgs) [inline how-to/about-how-to.md:45]: no spacing between "conceptually" and "Read".

File: how-to/about-how-to.md [hunk 4]
```diff
@@ -30,18 +30,21 @@ Creating a good overview comes down to thinking about who is going to be using i
...
 ```
 Make sure you meet the following prerequisites before following the steps:
 
-* API credentials for the {company} v3.5 API.
+* API credentials for the {company} v3.5 API. Refer to creating your API credentials.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is conceptually.Read about the RESTful API. 
 * (Optional) A development tool (IDE) that displays API responses formatted for readability.
+* You need to have a list of Favourites before you can manage them. Refer to “Create Favourites” for more details.
```
Comment (jaredmorgs) [inline how-to/about-how-to.md:47]: The Refer to text you use here differs from your previous example. This one uses quotes whereas the other ones are not quoted.

What suggested format do you want to consistently recommend in this guidance template?

Maybe clear up that the quoted text is a link back to the other section using a hyperlink?

Comment (jahnvigupta) [reply how-to/about-how-to.md:47]: @jaredmorgs  How can I make it clear that it is a hyperlink, changing colour to blue works?

**Pull request merged**